src/types/bytes.rs

```diff
@@ -67,6 +67,10 @@ impl PyBytes {
     /// Creates a new Python byte string object from a raw pointer and length.
     ///
     /// Panics if out of memory.
+    ///
+    /// # Safety
+    ///
+    /// Unsafe as it deferences the raw pointer `ptr`.
```

Comment:
Let's make this more explicit. 
https://doc.rust-lang.org/std/slice/fn.from_raw_parts.html#safety can be a good reference.
Or just noting a link to is sufficient.